openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -636,6 +638,15 @@ def delete_library(library_key):
         raise
 
 
+def _get_library_component_tags_count(library_key) -> dict:
+    """
+    Get the count of tags that are applied to each component in this library, as a dict.
+    """
+    # Create a pattern to match the IDs of the library components, e.g. "lb:org:id*"
+    library_key_pattern = str(library_key).replace("lib:", "lb:") + "*"
```

Comment:
Nit:
```suggestion
    library_key_pattern = str(library_key).replace("lib:", "lb:", 1) + "*"
```
Replace only the first occurrence and avoid problems if we have an org with "lib"